ginivision/src/main/res/layout-sw600dp-land/gv_fragment_noresults.xml

```diff
@@ -1,136 +1,217 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/gv_noresults_fragment_background"
-    android:fillViewport="true">
+    android:orientation="vertical">
 
-    <LinearLayout
+    <TextView
+        android:id="@+id/gv_no_results_header"
+        style="@style/GiniVisionTheme.NoResults.Header.TextStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignLeft="@+id/gv_no_results_headline"
+        android:layout_alignParentTop="true"
+        android:layout_alignStart="@+id/gv_no_results_headline"
```

Comment:
I'm not sure but I can remember that Android Studio was always "suggesting" to use `alignStart` instead of `alignLeft`. Was it because I was using a minimum SDK higher than the minimum required to use only `alignStart`?

Reply:
Yes, for backwards compatibility we have to use both.